### PR TITLE
Remove a MUST NOT

### PR DESCRIPTION
**Pull request by gorryfair**
Be firm, but avoid MUST NOT apply across different protocols
TO ADDRESS #914

### CODE REVIEW

File: draft-ietf-taps-interface.md [hunk 1]
```diff
@@ -1801,11 +1801,15 @@ Protocol Stack(s) backing a Connection. These Connection Properties may be
...
 Protocol Specific Properties are defined in a transport- and 
-implementation-specific way, and MUST NOT apply across different protocols. 
-Too much reliance by an application on Protocol Specific Properties can significantly 
-reduce the flexibility of a transport services implementation.
+implementation-specific way to
+permit more specialized protocol features to be used. 
+Too much reliance by an application on Protocol Specific Properties can significantly reduce the flexibility 
+of a transport services implementation to make appropriate
+selection and configuration choices. Therefore, it is RECOMMENDED that
+Protocol Properties are normnally used for properties common across different protocols and that
```
Comment (mwelzl) [inline draft-ietf-taps-interface.md:1811]: Typo in "normally", but: wouldn't this read better by altogether dropping the "normally"?

**Review by mwelzl — CHANGES_REQUESTED** [review]
just a small thing again, else lgtm

File: draft-ietf-taps-interface.md (hunk 1 above)
Comment (gorryfair) [reply draft-ietf-taps-interface.md:1811]: Much better without the word! (I will remove)

**Comment by gorryfair** [comment]
This was discussed at the Interim and final text approved by MW.

**Pull request merged**